azure-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2020 Microsoft and others.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+
+# Build your hawkBit Extensions and run tests with Apache Maven.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/java
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#maven
+
+pool:
+  vmImage: "ubuntu-18.04"
```

Comment:
Optional: What about `ubuntu-latest`?

Reply:
I would not do that. Latest may switch over night and who knows what impact that might have.

Reply:
Azure DevOps seems to cover that with a warning in the build log. Anyway fair enough!
```
##[warning] Ubuntu-latest pipelines will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816
```